Source/Scene/Cesium3DTileFeature.js

```diff
@@ -0,0 +1,262 @@
+     * handler.setInputAction(function(movement) {
+     *     var feature = scene.pick(movement.endPosition);
+     *     if (Cesium.defined(feature) && (feature.primitive === tileset)) {
+     *         var properties = tileset.properties;
+     *         if (Cesium.defined(properties)) {
+     *             for (var name in properties) {
+     *                 if (properties.hasOwnProperty(name)) {
+     *                     console.log(name + ': ' + feature.getProperty(name));
+     *                 }
+     *             }
+     *         }
+     *     }
+     * }, Cesium.ScreenSpaceEventType.MOUSE_MOVE);
+     */
+    function Cesium3DTileFeature(tileset, content, batchId) {
+        this._content = content;
+        this._batchId = batchId;
+        this._color = undefined;  // for calling getColor
+
+        /**
+         * All objects returned by {@link Scene#pick} have a <code>primitive</code> property.
+         *
+         * @type {Cesium3DTileset}
+         *
+         * @private
+         */
+        this.primitive = tileset;
+    }
+
+    defineProperties(Cesium3DTileFeature.prototype, {
+        /**
+         * Gets and sets if the feature will be shown.
+         *
+         * @memberof Cesium3DTileFeature.prototype
+         *
+         * @type {Boolean}
+         *
+         * @default true
+         */
+        show : {
+            get : function() {
+                return this._content.batchTable.getShow(this._batchId);
+            },
+            set : function(value) {
+                this._content.batchTable.setShow(this._batchId, value);
+            }
+        },
+
+        /**
+         * Gets and sets the highlight color multiplied with the feature's color.  When
+         * this is white, the feature's color is not changed.
+         * <p>
+         * Only <code>red</code>, <code>green</code>, and <code>blue</code> components
+         * are used; <code>alpha</code> is ignored.
+         * </p>
+         *
+         * @memberof Cesium3DTileFeature.prototype
+         *
+         * @type {Color}
+         *
+         * @default {@link Color.WHITE}
+         */
+        color : {
+            get : function() {
+                if (!this._color) {
+                    this._color = new Color();
+                }
+                return this._content.batchTable.getColor(this._batchId, this._color);
+            },
+            set : function(value) {
+                this._content.batchTable.setColor(this._batchId, value);
+            }
+        },
+
+        /**
+         * Gets the feature content.
+         *
+         * @memberof Cesium3DTileFeature.prototype
+         *
+         * @type {Cesium3DTileContent}
+         *
+         * @readonly
+         */
+        content : {
+            get : function() {
+                return this._content;
+            }
+        }
+    });
+
+    /**
+     * Returns whether the feature contains this property. This includes properties from this feature's
```

Comment:
Seems like this or somewhere on this reference doc page, we should link to https://github.com/AnalyticalGraphicsInc/3d-tiles/tree/master/TileFormats/BatchTable#batch-table-hierarchy